sycl/doc/extensions/SPIRV/SPV_INTEL_optnone.asciidoc

```diff
@@ -0,0 +1,116 @@
+SPV_INTEL_optnone
+=================
+
+Name Strings
+------------
+
+SPV_INTEL_optnone
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
```

Comment:
```suggestion
https://github.com/intel/llvm
```

Reply:
Why?

Reply:
We are not publishing it there so I don't understand why do we need to redirect questions there